src/http_server.cc

```diff
@@ -2564,6 +2564,12 @@ HTTPAPIServer::EVBufferToRawInput(
         v_idx++;
       }
 
+      if (base_size == 4 && strncmp(base, "Ping", base_size) == 0) {
```

Comment:
One question: do we need to make it flexible in case the ping format is changed in the future?